template/6-non_functional_requirements.md

```diff
@@ -2,22 +2,46 @@
+
+### Moderation
+
+Moderation is a crucial aspect of any public social media platform, and our application is no exception. Our policy regarding moderation will be based on a reporting system complemented by AI analysis and human judgment.
+
+If content is reported by multiple users, it will first be evaluated by an AI system to determine its appropriateness. If the AI model cannot fully decide on the nature of the content, human moderators will be engaged to make the final decision on whether to remove the content or not. This combined approach ensures that moderation is efficient, accurate, and fair, maintaining a safe and respectful environment for all users.
 
-- Privacy
-- Moderation
-- Authentication
 
 ## Adoptions, Scalability and Availability
 
-*What kind of traffic patterns do you expect to see?*
+### Adoption
+
+1. **Quick Engagement**: Upon the initial launch of the app, users should be able to view and post content with minimal steps to reduce conversion costs and facilitate quick engagement.
+2. **Challenge Tour**: When accessing the challenges page for the first time, users should receive a brief tour explaining how the challenges work and the logic behind Centauri points.
+3. **Location Awareness**: Users should be able to view nearby content on a map immediately, highlighting the location-based features of the application.
+
+### Scalability
+
+1. **High User Capacity**: The application must scale well to accommodate a high number of users and a large volume of content, ensuring seamless performance as the user base grows.
+2. **Content Density Performance**: It must also maintain good performance and responsiveness in high content density areas, such as big towns, monuments, and concerts, to provide a smooth user experience in these environments.
+3. **Feature Flexibility**: The backend should be designed to easily support the addition of new features over time, ensuring the application can evolve and adapt to meet changing user needs and technological advancements.
 
-*Are there known periods of bursty traffic that the MVP must be designed to support?*
+### Availability
 
-- High concentration of users in one place, e.g. concerts
+1. **High Uptime**: Uptime is crucial for any social media platform. Therefore, our application should maintain an uptime above 99.9% to ensure that users have reliable and continuous access to the service.
+2. **GPS Signal Handling**: In cases of poor or low precision GPS signal, the application should remain usable. It should include a manual refresh option to prevent content from appearing or disappearing unexpectedly, providing a more stable user experience.
```

Comment:
> It should include a manual refresh option to prevent content from appearing or disappearing unexpectedly, providing a more stable user experience.

This is the default behaviour of the app, right? In this case, it may make sense to reformulate it to:
"The feed should be refreshed manually, not on position change, to prevent content from appearing or disappearing unexpectedly, providing a more stable user experience."

What do you think? Again, doesn't this become too functional?

Reply:
I believe that is a good idea indeed.

Done in 27aeb271d0fca58fae43b56e7f5f48df0d684ebf.